README.md

```diff
@@ -2,16 +2,16 @@
 
 ## Usage
 
-Expects a `project.qgs` project file and all the files it depends on in the `/etc/qgisserver/`
-directory. Either you create another image to add those files or you inject them using
-a volume. For example:
+The docker container needs to have access to all files of the QGIS project to be published.
```

Comment:
```suggestion
The Docker container needs to have access to all files of the QGIS project to be published.
```

Reply:
Ping @anderso: would you mind fixing the `D` in the Docker name?